docs/learn-flink/etl.md

```diff
@@ -149,6 +149,7 @@ this would mean doing some sort of GROUP BY with the `startCell`, while in Flink
 {% highlight java %}
 rides
     .flatMap(new NYCEnrichment())
+    // .keyBy("fieldname") is @Deprecated, could use .keyBy(value -> value.startCell) 
```

Comment:
Why don't we just use the correct version instead? Again I can't see a reason why we should even mention deprecated methods.